docs/conf/portland/2018/tickets.rst

```diff
@@ -4,59 +4,82 @@
 Tickets
 =======
 
-**Tickets on sale now!**
+{% if flagticketsonsale %}
+
+**Tickets are on sale now!**
+
+{% else %}
```

Comment:
We should probably add a `flagticketssoldout` so this doesn't break when you generate a new conf.